src/main/java/com/google/devtools/build/lib/analysis/config/CoreOptions.java

```diff
@@ -650,7 +650,7 @@ public OutputDirectoryNamingSchemeConverter() {
 
   @Option(
       name = "experimental_allow_unresolved_symlinks",
```

Comment:
How about making this the old name and using just `allow_unresolved_symlinks` as the new name? Having to disable a default option via a flag called `experimental` could be confusing for users.

Reply:
I am not sure the flag will stick around or whether people should actively toggle it; if it doesn't stick around then we should keep the `experimental_` prefix. 

 And I am not the person to make that call - I am just doing the bare minimum to enable this in Bazel 6.0.

Reply:
Realistically, I don't think we will flip this flag at Google anytime soon so I guess it'll have to stay?

Reply:
For me the question is more whether we want people (both internally and externally) to change their default value. If not, let's keep the prefix?

Reply:
I was scratching my head as to how to decide that. Your line of reasoning is quite nice, so I guess let's keep it?